### ISSUE TITLE
Should we drop the Facility Locator tooltip?

### ISSUE THREAD

**Issue by webinista** [department-of-veterans-affairs/vets-website#2083]
The Section 508 report we received noted that we don't have a label on the location field. Using the placeholder text as the label text solves that issue, but adds redundant text for assistive tech users. 

"Ghost text" isn't as effective as adjacent explanatory text, but our explanatory text is hidden behind a tooltip. The tooltip adds CSS and JavaScript overhead, and isn't as accessible or easy-to-use as text without a tooltip. Removing the tooltip would lighten page weight a bit, make that form more accessible, and is easier to maintain over the long term. 

Blocks / related to #2067 #2069/#2081

@maryannbrody @emilyville @gnakm Please weigh in and tag others who may know have opinions about this.

**Comment by Drakenhart** [comment]
Note: I do not know specifically how you are handling your tool tips.

But, (fyi) Tool tips can be made accessible and useful if you are exposing the title tag rather then creating something with just javascript.

http://accessibility.athena-ict.com/aria/examples/tooltip.shtml

I hope this helps.
~~~

Secondly any form field needs a label in general, even if it's not visually exposed.
https://www.w3.org/WAI/tutorials/forms/instructions/
https://www.w3.org/WAI/tutorials/forms/ (ignore the tutorial part of this, this contains some important notes on Accessibility for forms in general.)

http://webaim.org/techniques/forms/
http://www.afb.org/info/accessibility/creating-accessible-websites/accessible-forms/235
http://htmldog.com/guides/html/advanced/forms/


We've run into issues using forms and how screen readers interact with them, even simple one-item form inputs. Forms are tricky beasts, to be sure.

**Comment by webinista** [comment]
Thanks for this, @Drakenhart. This is useful to have as we add more forms to the site. In this case, however, I'm asking:

1. Is a tooltip is the best way to display instructional text to the user?
2. Is it worth the additional CSS and JavaScript overhead?

**Comment by maryannbrody** [comment]
@webinista, my $0.02 -- 

What if we removed the tooltips, and had a tighter visual connection between pins & text. For example, maybe the pins become some other indicator (circles) and as a user clicks through the locations via the menu on the right, the pin moves to indicate location. 

Not sure if this helps from a technical perspective, but it could potentially get rid of the tool tips. Google maps functions more like this where various locations have a uniform icon, and the pin jumps from icon to icon as the user clicks the text.

**Comment by webinista** [comment]
@maryannbrody To clarify: the pins and text are Google Maps-controlled. I think we can change the pin color, but most everything else is on Google's servers. 

By "tool tip," I am talking about the help widget next to the search form. It's called `searchtooltip` in the markup, so I've been using "tool tip" without really explaining what I'm talking about.

![screen shot 2016-04-22 at 2 53 03 pm](https://cloud.githubusercontent.com/assets/354277/14755851/f5cd8a6e-0899-11e6-914f-28dca79ca564.png)

**Comment by maryannbrody** [comment]
@webinista - Got it!

In that case, I agree that ghost text is not the best solution (even as it's currently implemented). Sometimes is usability studies you'll see users click "search" without typing anything into the box when there's ghost text present. My vote would be to pull the helper text out of the tool tip and instead display it above the search box. cc: @emilyville

**Comment by Drakenhart** [comment]
It's a very good question, so I went on a web walk about from major e-commerce sites (who's searches are used daily) to doctor and hospital search sites (for those with multiple locations/offices).

None use tool tips the way it's being used in the example above. All use labels (though about half of them have the labels hidden). it's 50/50 also for having anything inside of the search input.

Some usability articles I've run across suggest to not complicate the search input interface, as people at this point know what it's for and how to use it. However it is a widely used style (ghost text).

My suggestions: 

* Get rid of the tool tips.
* Use a label that is action oriented. (Like "Enter zip code, city, state, U.S. Territory, or address".)
* Make the label visible.

If you insist on using ghost text:
**Tested this in Chrome Vox** If you repeat the same words of the label inside the value attribute of the input field, it won't repeat.

**Comment by gnakm** [comment]
Hi all --

Sorry that I am late to this conversation. 
For consistency sake, I think we should not have a tooltip. Piggy backing on @Drakenhart 's idea, throughout our site, we add a label with parenths like the one below:
<img width="571" alt="screen shot 2016-04-25 at 11 55 13 am" src="https://cloud.githubusercontent.com/assets/13420618/14795563/eb654f1c-0add-11e6-9c54-33d40f6e91db.png">

We could do something like this:
<img width="496" alt="screen shot 2016-04-25 at 12 04 35 pm" src="https://cloud.githubusercontent.com/assets/13420618/14795641/3163be4a-0ade-11e6-923a-6e56086fec5c.png">

**Comment by maryannbrody** [comment]
I agree with @gnakm - this is what I was thinking too.

**Comment by emilyville** [comment]
I concur! we only have a few tooltip instances (here and GI tool) and frankly, I think there's better ways to deal with this information from a UX perspective with less front end overhead.

**Comment by webinista** [comment]
Removed the tool tip, and added an off screen label. It's there for assistive tech users, but visually hidden. I also changed the color of the placeholder text so that there's more contrast between the foreground and background.  This is what it looks like now.

![screen shot 2016-05-02 at 1 12 34 pm](https://cloud.githubusercontent.com/assets/354277/14966492/92ac962a-1067-11e6-8495-306b8f5cfc87.png)
